<launch>

 <node name="msf_position_sensor" pkg="msf_updates" type="position_pose_sensor" clear_params="true" output="screen">
       <remap from="msf_core/imu_state_input" to="/fcu/imu" />
       <remap from="msf_updates/position_input" to="/noise_drift_handler/position_output" />
       <rosparam file="$(find msf_updates)/params/pp_positiononly_VH/msf_sensor_fix.yaml"/>
       <param name="capability_group" value="MSF"></param>
 </node>
 
    <node name="pos_to_error" pkg="msf_updates" type="pos_to_error.py" output="screen">
        <remap from="pos_to_error/datainput" to="msf_core/state_out"/>
        <remap from="pos_to_error/truthinput" to="/vicon/firefly_sbx/firefly_sbx"/>
        <rosparam file="$(find msf_updates)/params/pp_positiononly_VH/pos_to_error_fix.yaml"/>
    </node>
    
    <node name="noise_drift_handler" pkg="msf_updates" type="noise_drift_handler.py" output="screen">
        <remap from="noise_drift_handler/transform_input" to="/vicon/firefly_sbx/firefly_sbx"/>
        <rosparam file="$(find msf_updates)/params/pp_positiononly_VH/noise_drift_handler_fix.yaml"/>
    </node>
</launch>
